src/model/MutableClimbDataSource.ts

```diff
@@ -107,6 +107,28 @@ export default class MutableClimbDataSource extends ClimbDataSource {
 
     const newDocs: ClimbChangeDocType[] = []
 
+    // when adding new climbs, ensure they are added in expected sort order, such that:
+    // * multiple climbs keep the order they're input in.
+    // * all newly added climbs come after existing climbs.
+    const maxExisting = (await this.climbModel
+      .find({ _id: { $in: parent.climbs } })
+      .select('metadata.left_right_index')
+      .sort({ 'metadata.left_right_index': -1 })
+      .limit(1))[0]?.metadata.left_right_index
+    let newClimbLeftRightIndex = (maxExisting ?? 0) + 1
+
+    function resolveLeftRightIndex (i: number): { left_right_index: number } | null {
+      // user input is always prioritized
+      if (userInput[i].leftRightIndex !== undefined) {
```

Comment:
can you use `!=` operator to account for nullness?

Reply:
No problem, resolved ✔️ 
Missed this convention, have historically used eslint which prefers triple equal (https://eslint.org/docs/latest/rules/eqeqeq).

I guess in this case, while the type declaration `leftRightIndex?: number` tells us it'll be a `number | undefined`, we don't actually validate that since data comes in as `any` from graphql, so client is free to send `null` and we should handle that as well